linux/scripts/start_seattle.sh

```diff
@@ -27,10 +27,16 @@ sleep 1
 #     'ax': shows all processes
 #     'ww': makes sure that the output is not limited by column length.
 #     
-
-NMMAIN=`ps axww 2>/dev/null | grep nmmain.py | grep -v grep`
-SOFTWAREUPDATER=`ps axww 2>/dev/null | grep softwareupdater.py | grep -v grep`
-
+# However, OpenWrt has a Busybox implementation of 'ps' which does not provide
+# the used parameters. Therefore we use 'ps ww' on OpenWrt instead.
+if [ "$(uname -m)" == "mips" ];
+then
+    NMMAIN=`ps ww 2>/dev/null | grep nmmain.py | grep -v grep`
```

Comment:
You could also just (re)define a variable, say `PS_COMMAND`, in the if/else construct, and later do `$PS_COMMAND` so as to use the rest of the command string unchanged.